src/tests/presets/test_universal_deployer.cairo

```diff
@@ -1,23 +1,18 @@
 use openzeppelin::presets::universal_deployer::UniversalDeployer::ContractDeployed;
 use openzeppelin::presets::universal_deployer::UniversalDeployer;
-use openzeppelin::tests::mocks::erc20_mocks::DualCaseERC20Mock;
 use openzeppelin::tests::utils::constants::{NAME, SYMBOL, SUPPLY, SALT, CALLER, RECIPIENT};
+use openzeppelin::tests::utils::events::EventSpyExt;
 use openzeppelin::tests::utils;
 use openzeppelin::token::erc20::interface::{IERC20Dispatcher, IERC20DispatcherTrait};
 use openzeppelin::utils::deployments::{DeployerInfo, calculate_contract_address_from_udc};
 use openzeppelin::utils::interfaces::{
     IUniversalDeployerDispatcher, IUniversalDeployerDispatcherTrait
 };
 use openzeppelin::utils::serde::SerializedAppend;
-use starknet::ClassHash;
-use starknet::ContractAddress;
-use starknet::testing;
+use snforge_std::{EventSpy, spy_events, declare, start_cheat_caller_address};
+use starknet::{ClassHash, ContractAddress};
 
 
-fn ERC20_CLASS_HASH() -> ClassHash {
```

Comment:
Why remove the helper if the class hash is used twice below?

Reply:
No strong reason. Will update